app/virtual_gamepad_hub.coffee

```diff
@@ -5,12 +5,15 @@ Virtual gamepad hub class
 
 gamepad = require './virtual_gamepad'
 log = require '../lib/log'
+config = require '../config'
+
+num_gamepads = config.ledBitFieldSequence.length
 
```

Comment:
How about I add the following comment here.
```coffeescript
# num_gamepads should be an array of 'bit fields'.
# the bit fields are numbers from 0-15, resulting in the following LED arrangements:
#  0 - ....
#  1 - *...
#  2 - .*..
#  3 - **..
#  4 - ..*.
#  5 - *.*.
#  6 - .**.
#  7 - ***.
#  8 - ...*
#  9 - *..*
# 10 - .*.*
# 11 - **.*
# 12 - ..**
# 13 - *.**
# 14 - .***
# 15 - ****
```

Reply:
I'll take the comment for sure. I will then accept your PR (after I tested it, assuming it works). But I think I'll also add some info in the README any maybe a separate file, explaining `config.json`'s options would be a good thing to have anyway. So I'll probably add that as well.

Reply:
Sounds good. It's a shame JSON doesn't support comments.